pkgs/development/tools/misc/blackfire/php-probe.nix

```diff
@@ -18,52 +18,52 @@ let
     "8.1" = "blackfire-20210902";
   }.${phpMajor} or (throw "Unsupported PHP version.");
 
-  version = "1.78.1";
+  version = "1.84.0";
 
   hashes = {
     "x86_64-linux" = {
       system = "amd64";
-      sha256 = "Q9VuZewJ/KX2ZL77d3YLsE80B0y3RYg/hE2H14s9An4=";
+      sha256 = "4tAqe1ev2s4ZwzPptgXuVL4ZXF37ieGyonBxOFMUKTs=";
```

Comment:
Next time please don't strip prefixes from sri hashes.

Reply:
With sri, you mean the prefix `sha256-` right?

Reply:
yes